config/application.rb

```diff
@@ -34,5 +34,9 @@ class Application < Rails::Application
     config.action_mailer.notify_settings = {
       api_key: ENV["GOVUK_NOTIFY_API_KEY"]
     }
+
+    # Used to handle HTTP_X_WITH_SERVER_DATE header for server side datetime overwrite
+    require "middleware/time_traveler"
```

Comment:
Can we move this to the top of the file as its more common location

Reply:
You get an error if you do this, I think because we're in the `application.rb` file the Rails auto loader hasn't setup the load paths for our application files yet (but it will by the time its evaluating the class). You can `require_relative` at the top of the file, but then you have to put the full `../lib/middleware` path which didn't feel as nice as this.

Reply:
Ah ok, thats fine 👍